datahub-web-react/src/conf/analytics.ts

```diff
@@ -3,9 +3,9 @@ const config: any = {
     // googleAnalytics: {
     //    trackingId: 'UA-24123123-01',
     // },
-    // mixpanel: {
-    //    token: 'fad1285da4e618b618973cacf6565e61',
-    // },
+    mixpanel: {
+        token: '9720ed0d307ff94c9977ced8fe0c5f2a',
```

Comment:
I believe we can change that to an anchor we populate on CI, so if there is a QA version at some point we can manage that and not hard code it here.
```suggestion
        token: 'replace-me',
```